### ISSUE TITLE
genre Indicators in titles

### ISSUE THREAD

**Issue by galleron** [distantreading/WG1#40]
Carolin asked:
"are genre indicators also treaten as "otherEntities"?
if no, we need more columns for genreIndicatorRole and genreIndicatorSyntax
for things like "A Tale of Manchester Life"

My proposal:
So far, we have worked with the idea that genre indicators are not otherEntities. In the example « A Tale of Manchester Life », I would annotate « Tale » as genreIndicator; Manchester as place, and Life as « otherEntity ».
I will aadd the extra colums in the guidelines, accordingly.

**Comment by jberenike** [comment]
I’ve treated seriality indications like „zweiter Band“ (second vol.) 
as "otherEntity.“ (not as genreIndicator).

"Nürnberg. Zweiter Band“

**Comment by RoxanaPatras** [comment]
I treated "life", "adventures", "roman", "(hi)story", "volume" - embedded in either first part or 2nd part of a title - as  genreEntity (see Genette's remarks on the generic context of titles in Seuils) and not as otherEntity. 

If necessary for this particular article, I'd add columns for every category that seems incomplete. If not, I'd argumentate the row - and it will argumentate a lot! - only after we're done with persons and places.   

 Don't know if this has been documented properly, but for this article we'll focus only on personEntities and placeEntities. This is something me, Carolin & Ioana agreed on after considering the following argument which popped out after our first round of annotations on 6 collections: 1. There are categories more volatile, heterogeneous & dynamic (otherEntities, genreEntities) & there are categories more stable (places & persons). 

I think that we must ponder more on the way we annotate:
a. genres: if I were to chose, instead of "novel" I'd annotate "a dramatic novel" or instead of "story", "Bessarabian story".
b. otherEntities, which is a dustbin category rightnow: we put there everything that's not person, place or genre. 
c. +++ attribute lexemes: "purloined" in "purloined letter" :-))

**Comment by galleron** [comment]
Agreed on the fact that "zweite band" is "other entity", since we do not have a proper genre indicator here (second par t of what? A novel? A story? etc.)
We need, indeed, to focus on persons and places, but having complete data is always good!

**Comment by kmihurko** [comment]
I tried to copy the indicator entity (zgodovinski roman = historical novel) in my table but is just possible to enter yes or no. Am I doing something wrong?

**Comment by kmihurko** [comment]
And another question: all genre indicators in Slovenian corpus are actually the subtitles, so they are not in the titles in the table. Is it OK to treat them as genre indicators?

**Comment by RoxanaPatras** [comment]
Hi, Katja! This is probably the case with many collections. However, we considered that a subtitle is a part of the title. Back to Genette and his Seuils: short titles (i.e. without subtitles) should be considered "fake titles". Thus it's better to have the complete picture.

**Comment by RoxanaPatras** [comment]
> I tried to copy the indicator entity (zgodovinski roman = historical novel) in my table but is just possible to enter yes or no. Am I doing something wrong?

For the moment, we haven't decided to take "historical novel" but only "novel". 

I agree with you that "historical novel" serves better our inquiry on genre indicators; however, in the annotation guidelines we have specified that only 1 word (usually the noun) should be pasted.

**Comment by kmihurko** [comment]
Thank you ... but if I want to enter the word novel it is impossible, it says: A user has restricted values that can be entered into this cell ...

**Comment by RoxanaPatras** [comment]
> Thank you ... but if I want to enter the word novel it is impossible, it says: A user has restricted values that can be entered into this cell ...

First, you say if there's a genre indicator: yes/ no

The, in the next column, you say genreIndicator: "novel"

**Comment by kmihurko** [comment]
Do you mean column AA?

**Comment by RoxanaPatras** [comment]
> And another question: all genre indicators in Slovenian corpus are actually the subtitles, so they are not in the titles in the table. Is it OK to treat them as genre indicators?

To make it more clear: you need "to fish" the original titles from titlepages (if there's such a div in the front of the TEI file) or from Mr. Google's references to original editions. Hope this helps!

**Comment by RoxanaPatras** [comment]
> Do you mean column AA?

Probably! I don't have the excel sheet under my eyes right now, but you could download the It collection and check which one it is :-))

**Comment by kmihurko** [comment]
Yes, no problem to find subtitles, but still the problem how to copy them in the column AA ...

**Comment by RoxanaPatras** [comment]
> Yes, no problem to find subtitles, but still the problem how to copy them in the column AA ...

Ok. Let me see what I can do about it.

**Comment by jberenike** [comment]
> Agreed on the fact that "zweite band" is "other entity", since we do not have a proper genre indicator here (second par t of what? A novel? A story? etc.)
> We need, indeed, to focus on persons and places, but having complete data is always good!

ok, so second vol. etc. are otherEntity, not Genre.

**Comment by jberenike** [comment]
> > And another question: all genre indicators in Slovenian corpus are actually the subtitles, so they are not in the titles in the table. Is it OK to treat them as genre indicators?
> 
> To make it more clear: you need "to fish" the original titles from titlepages (if there's such a div in the front of the TEI file) or from Mr. Google's references to original editions. Hope this helps!

this is important! We thus need to scan the original XML-files from ELTeC to check whether a sub title is missing. Is that what you're saying? I did have the case!  The spreadsheet just mentioned the main title, not the subtitle.

**Comment by RoxanaPatras** [comment]
> > > And another question: all genre indicators in Slovenian corpus are actually the subtitles, so they are not in the titles in the table. Is it OK to treat them as genre indicators?
> > 
> > 
> > To make it more clear: you need "to fish" the original titles from titlepages (if there's such a div in the front of the TEI file) or from Mr. Google's references to original editions. Hope this helps!
> 
> this is important! We thus need to scan the original XML-files from ELTeC to check whether a sub title is missing. Is that what you're saying? I did have the case! The spreadsheet just mentioned the main title, not the subtitle.

Yes, this is it. We need to have as complete titles as possible. When we did the first round of annotations we discovered very different titling practices around Eltec collections (fake titles, complete titles, later titles, etc), so we decided to stick - as much as possible - to original titles. I checked all titles in the IT collection (both in the xml files and on various catalogues) and I found a few surprising things (they're marked in red). Hope this helps!

**Comment by lb42** [comment]
Out of curiosity I ran a quick check to see which ELTeC corpora have 
titlepages encoded explicitly in the XML source. The results are (to say 
the least) variable. But I can produce a nice file containing just the 
text identifier and the text of the titlepage (if it's present) for you 
if you like.

**Comment by lb42** [comment]
Also, if you find errors or omissions in the xml sources, it wod be nice if these were fixed either in the source xml files directly or by noting the problem in a ticket in the appropriate repository!

**Comment by jberenike** [comment]
> Also, if you find errors or omissions in the xml sources, it wod be nice if these were fixed either in the source xml files directly or by noting the problem in a ticket in the appropriate repository!

yes, doing that for deu. Thanks for encouragment! For deu, I point back to TitleStudy: spreadsheet deu_jbh will have all missing sub titles marked in red. (as for IT sample)

**Comment by lb42** [comment]
Here are the counts for currently available titlepages:
cze 16 texts 16 fronts  14 titlepages 
deu 98 texts 3 fronts  0 titlepages 
eng 100 texts 95 fronts  94 titlepages 
fra 100 texts 56 fronts  33 titlepages 
hun 100 texts 17 fronts  1 titlepages 
ita 34 texts 27 fronts  0 titlepages 
nor 27 texts 0 fronts  0 titlepages 
por 98 texts 97 fronts  96 titlepages 
rom 70 texts 68 fronts  63 titlepages 
slv 100 texts 0 fronts  0 titlepages 
spa 45 texts 40 fronts  37 titlepages 
srp 49 texts 46 fronts  46 titlepages 

If you'd like to see the titlepages in plain text, let me know for which repo!

**Comment by RoxanaPatras** [comment]
> Here are the counts for currently available titlepages:
> cze 16 texts 16 fronts 14 titlepages
> deu 98 texts 3 fronts 0 titlepages
> eng 100 texts 95 fronts 94 titlepages
> fra 100 texts 56 fronts 33 titlepages
> hun 100 texts 17 fronts 1 titlepages
> ita 34 texts 27 fronts 0 titlepages
> nor 27 texts 0 fronts 0 titlepages
> por 98 texts 97 fronts 96 titlepages
> rom 70 texts 68 fronts 63 titlepages
> slv 100 texts 0 fronts 0 titlepages
> spa 45 texts 40 fronts 37 titlepages
> srp 49 texts 46 fronts 46 titlepages
> 
> If you'd like to see the titlepages in plain text, let me know for which repo!

Wow! Thanks, Lou!

**Comment by dianamsmpsantos** [comment]
I am sorry to say that not al title pages of the Portuguese collection do
conform to the actual titlepage of the edition used. I was planning to
"solve" this at a later time. Please let me know what exactly do you need
to be in the titlepage (in addition to the title) for your study so that I
can go through the collection and put it there.
Diana

Roxana PATRAS <notifications@github.com> escreveu no dia sábado, 30/05/2020
à(s) 16:38:

**Comment by galleron** [comment]
We have now two issues in this ticket:
1° the incompleteness of titles in ELTeC collections: we have already observed this in the first round of annotation, and indeed we looked at complete titles for the Romanian, French and English collection.  Since collections are quite short, it can be checked quite easily directly in the excel file. 
2° Genre indicator: my mistake here, I'm sorry! We were proceding as Roxana says in the first round (column AA: yes/no, then column AB the actual indicator), but this is one of the columns that was left aside so as to simplify the encoding. You can force excel to accept the genre indicatir noun instead of "yes", so please do so.

**Comment by jberenike** [comment]
> Here are the counts for currently available titlepages:
> cze 16 texts 16 fronts 14 titlepages
> deu 98 texts 3 fronts 0 titlepages
> eng 100 texts 95 fronts 94 titlepages
> fra 100 texts 56 fronts 33 titlepages
> hun 100 texts 17 fronts 1 titlepages
> ita 34 texts 27 fronts 0 titlepages
> nor 27 texts 0 fronts 0 titlepages
> por 98 texts 97 fronts 96 titlepages
> rom 70 texts 68 fronts 63 titlepages
> slv 100 texts 0 fronts 0 titlepages
> spa 45 texts 40 fronts 37 titlepages
> srp 49 texts 46 fronts 46 titlepages
> 
> If you'd like to see the titlepages in plain text, let me know for which repo!

thank you! What's the difference between fronts and titlepages? And why doesn't DEU have any titlepages? (sorry for basicness of question)

**Comment by lb42** [comment]
The `<front>` element in a transcription contains everything before the `<body>` ... i.e. authorial prefaces, dedications, etc. as well as  the title page. DEU doesn't have any titlepages because whoever transcribed the texts decided they didn't want to include anything except the body of the text Sorry for the obviousness of the reply!

**Comment by jberenike** [comment]
this work has shown that the DEU collection has not been well curated for the subtitles! Researching the original subtitles for the particular edition is a substantial task --- I have invested several hours, if not days, because otherwise annotation would not have made sense. However, there is a degree of uncertainty involved in my additions as I used diverse resources on the web and not the original printed edition.

**Comment by jberenike** [comment]
> The `<front>` element in a transcription contains everything before the `<body>` ... i.e. authorial prefaces, dedications, etc. as well as the title page. DEU doesn't have any titlepages because whoever transcribed the texts decided they didn't want to include anything except the body of the text Sorry for the obviousness of the reply!

thank you! this helps.

**Comment by lb42** [comment]
Actually, @jberenike, it's a little unfair to say that the DEU collection is "not well curated ". If you look in the sourceDesc for each text there is a pretty good bibliographic description, including the full title. For example, here's DEU003's:
~~~~
 <bibl type="printSource"><author>Otto Ruppius</author>: <title>Der Pedlar. Roman aus dem
      amerikanischen Leben</title>, <pubPlace>Leipzig</pubPlace>:<publisher> Philipp Reclam
      jun.</publisher>, [um <date>1910</date>]. </bibl>
~~~~
As far as I can see your project chose to look at the titleStmt in the TEI Header. This contains only a short title for the ELTeC version itself, not the title of the original source at all.

**Comment by jberenike** [comment]
okay, this is helpful, Lou! Thanks for taking the time to point this out.
*My apologies* if I was to quick and did not choose my words carefully enough! ( mind you: I did not say "in general", but just "for the subtitles!" )
However, I understand now that our study's spreadsheet column was generated on the titleStmt, and not on sourceDesc/bibl[@type='printSource']/title  (I did not know that.)
 In any case: Sorry for the misunderstanding. 
(I was trying to get the annotation work done and kept on discovering that subtitles were missing , which I added by hand, which has taken soo much time than anticipated)

**Comment by lb42** [comment]
No need to apologise -- I didn't realise it till I looked more closely either! And this work has shown up several issues which need to be addressed in this collection. I just found for example that DEU004 is the only one which doesn't have a full biblio description, perhaps because it's only volume 2 of "Nurnberg",

**Comment by lb42** [comment]
p.s. I suppose it's too late, but here is the full list of titles as given in sourceDescs for the German repo./ 
[titles.txt](https://github.com/distantreading/WG1/files/4708394/titles.txt)

**Comment by jberenike** [comment]
> p.s. I suppose it's too late, but here is the full list of titles as given in sourceDescs for the German repo./
> [titles.txt](https://github.com/distantreading/WG1/files/4708394/titles.txt)

that's wonderful! I was just typing a message asking for this info.  will use it for DEU057 and up! Maybe the other collections can still use this kind of info too?

**Comment by jberenike** [comment]
> p.s. I suppose it's too late, but here is the full list of titles as given in sourceDescs for the German repo./
> [titles.txt](https://github.com/distantreading/WG1/files/4708394/titles.txt)

if possible, could you maybe even separate the elements by tapstops? (I'd like to copy paste the title and subtitle into one column of the spreadsheet).

**Comment by lb42** [comment]
Or would you prefer a file like this which has only the titles?
[titlesOnly.txt](https://github.com/distantreading/WG1/files/4708550/titlesOnly.txt)

**Comment by lb42** [comment]
I can easily run this on the other repositories. But they vary in the way they encode the bibliographic info. Some of them don't separate out the title from the other parts at all, except by using conventional punctuation marks. Here are a few examples: 
[ita-titles.txt](https://github.com/distantreading/WG1/files/4708580/ita-titles.txt)
[srp-titles.txt](https://github.com/distantreading/WG1/files/4708582/srp-titles.txt)
[rom-titles.txt](https://github.com/distantreading/WG1/files/4708584/rom-titles.txt)
[fra-titles.txt](https://github.com/distantreading/WG1/files/4708585/fra-titles.txt)

**Comment by DmytroYesUA** [comment]
Just a short remark regarding the diversity of genre titles. Literary scholars and readers might feel sorry to lose the charm of certain names (I do). Examples from the Polish collection that seems to make a difference: whether we take “romance” instead of “the afterlife romance” [romans zagrobowy], “novel” instead of “contemporary novel” / “historical novel”? Such genre definitions of texts for some reasons were chosen by their authors and can reflect interesting trends in the 'genre development' in particular literatures or several literatures (for instance: why did writers referred to the past in certain periods and rather to the present events - in others; maybe there is a similarities between, let's say French and Polish titles in particular decades?). For now, I leave the longer versions of titles but of course, shall be ready to adapt annotation to the finally defined scheme

**Comment by jkrybicki** [comment]
Excellent point, Dmytro!

Thanks,

Jan

 

Od: DmytroYesUA <notifications@github.com> 
Wysłano: poniedziałek, 1 czerwca 2020 01:29
Do: distantreading/WG1 <WG1@noreply.github.com>
DW: Subscribed <subscribed@noreply.github.com>
Temat: Re: [distantreading/WG1] genre Indicators in titles (#40)

 

Just a short remark regarding the diversity of genre titles. Literary scholars and readers might feel sorry to lose the charm of certain names (I do). Examples from the Polish collection that seems to make a difference: whether we take “romance” instead of “the afterlife romance” [romans zagrobowy], “novel” instead of “contemporary novel” / “historical novel”? Such genre definitions of texts for some reasons were chosen by their authors and can reflect interesting trends in the 'genre development' in particular literatures or several literatures (for instance: why did writers referred to the past in certain periods and rather to the present events - in others; maybe there is a similarities between, let's say French and Polish titles in particular decades?). For now, I leave the longer versions of titles but of course, shall be ready to adapt annotation to the finally defined scheme

**Comment by kmihurko** [comment]
I had same thoughts as Dmytro when annotating Slovenian titles and I completely agree that for literary scholars it is very important to have a full description . However, following Roxana's instructions I just put novel, tale etc.  I think we should all follow same criteria otherwise it seems that some collections are "poorer" than others. I can add missing words, just let me know ...

**Comment by RoxanaPatras** [comment]
> Just a short remark regarding the diversity of genre titles. Literary scholars and readers might feel sorry to lose the charm of certain names (I do). Examples from the Polish collection that seems to make a difference: whether we take “romance” instead of “the afterlife romance” [romans zagrobowy], “novel” instead of “contemporary novel” / “historical novel”? Such genre definitions of texts for some reasons were chosen by their authors and can reflect interesting trends in the 'genre development' in particular literatures or several literatures (for instance: why did writers referred to the past in certain periods and rather to the present events - in others; maybe there is a similarities between, let's say French and Polish titles in particular decades?). For now, I leave the longer versions of titles but of course, shall be ready to adapt annotation to the finally defined scheme

This is what I keep on saying for half a year now :-)) : that we need those genre attributes!!! But I'm prepared to take my "revenge" in the second stage of our little project, when we'll try to implement some Genettian categories of title analysis. Count on me, Dmytro!

**Comment by RoxanaPatras** [comment]
> I can easily run this on the other repositories. But they vary in the way they encode the bibliographic info. Some of them don't separate out the title from the other parts at all, except by using conventional punctuation marks. Here are a few examples:
> [ita-titles.txt](https://github.com/distantreading/WG1/files/4708580/ita-titles.txt)
> [srp-titles.txt](https://github.com/distantreading/WG1/files/4708582/srp-titles.txt)
> [rom-titles.txt](https://github.com/distantreading/WG1/files/4708584/rom-titles.txt)
> [fra-titles.txt](https://github.com/distantreading/WG1/files/4708585/fra-titles.txt)

Thanks, Lou! I know the Romanian collection by heart :-)) For Italian, all right, it helps - I'll run a recheck.

**Comment by RoxanaPatras** [comment]
> > I can easily run this on the other repositories. But they vary in the way they encode the bibliographic info. Some of them don't separate out the title from the other parts at all, except by using conventional punctuation marks. Here are a few examples:
> > [ita-titles.txt](https://github.com/distantreading/WG1/files/4708580/ita-titles.txt)
> > [srp-titles.txt](https://github.com/distantreading/WG1/files/4708582/srp-titles.txt)
> > [rom-titles.txt](https://github.com/distantreading/WG1/files/4708584/rom-titles.txt)
> > [fra-titles.txt](https://github.com/distantreading/WG1/files/4708585/fra-titles.txt)
> 
> Thanks, Lou! I know the Romanian collection by heart :-)) For Italian, all right, it helps - I'll run a recheck.

However, I today it's favors' day: Lou, could you please update the plot for the Rom collection? I think I should add 4 more titles in the annotation table.

**Comment by RoxanaPatras** [comment]
> I had same thoughts as Dmytro when annotating Slovenian titles and I completely agree that for literary scholars it is very important to have a full description . However, following Roxana's instructions I just put novel, tale etc. I think we should all follow same criteria otherwise it seems that some collections are "poorer" than others. I can add missing words, just let me know ...

To make it clear: all collections are very rich and all people who worked so as to make ElTeC real need to be appreciated. About attributes of genre - see my comments above, pls! - it's not my choice, it's for an optimal first extraction of data. 

We sometimes need to make compromises between what you want to do and what you actually can do.

**Comment by jberenike** [comment]
> Or would you prefer a file like this which has only the titles?
> [titlesOnly.txt](https://github.com/distantreading/WG1/files/4708550/titlesOnly.txt)

thank you! it would be better though to have the complete info with tapstops or semicola! In titlesOnly.txt some lines appear to contain just genre or publication indicator

**Comment by lb42** [comment]
On 01/06/2020 09:21, Roxana PATRAS wrote:
> However, I today it's favors' day: Lou, could you please update the 
> plot for the Rom collection? I think I should add 4 more titles in the 
> annotation table. 


Not sure what you mean. I don't think I should update your annotation 
table. The mosaic and index.html at 
https://distantreading.github.io/ELTeC/rom/ were last updated on 30/5 
and show a total of 70 titles. If that's wrong, maybe you forgot to 
check some new titles into the repo?

**Comment by lb42** [comment]
This is more of a discussion thread than an issue -- lots of different things mixed together. But fwiw here's my take on genre labelling.
1. It is important and interesting to know the genre labels assigned to a text by its author and its original readers (and maybe also by its readership over time, but that's more difficult to trace)
2. It is (probably) interesting to assign modern day labelling too, especially if it is a matter of categorising by some sort of objective criteria or using a formal system (Genette or whoever) 
3. THE TWO ARE COMPLETELY DIFFERENT!

**Comment by jberenike** [comment]
> > Or would you prefer a file like this which has only the titles?
> > [titlesOnly.txt](https://github.com/distantreading/WG1/files/4708550/titlesOnly.txt)
> 
> thank you! it would be better though to have the complete info with tapstops or semicola! In titlesOnly.txt some lines appear to contain just genre or publication indicator

okay, just checked again: some many of the text titles don't appear in the titles.txt. They just feature the book title.
E.g. DEU006: Georg Weerth: Sämtliche Werke in fünf Bänden.

In any case, I do find some of the missing info.

**Comment by lb42** [comment]
@jberenike says "thank you! it would be better though to have the complete info with tapstops or semicola! In titlesOnly.txt some lines appear to contain just genre or publication indicator"

The titlesOnly.txt contains whatever is tagged in the source as `<title>` -- the tagging may be wrong, or missing though.  So introducing "tapstops" wouldn't help much in the case where there is a `<title>` but no `<author>`. I'm afraid you have to  rely on either the XML tagging or the punctuation, and there's no sense in introducing an additional layer of pseudo-markup.

**Comment by lb42** [comment]
Here's the titles only file for SLV, for comparison (This one has some works with multiple titles)
[slv-titles.txt](https://github.com/distantreading/WG1/files/4710415/slv-titles.txt)

**Comment by jberenike** [comment]
> @jberenike says "thank you! it would be better though to have the complete info with tapstops or semicola! In titlesOnly.txt some lines appear to contain just genre or publication indicator"
> 
> The titlesOnly.txt contains whatever is tagged in the source as `<title>` -- the tagging may be wrong, or missing though. So introducing "tapstops" wouldn't help much in the case where there is a `<title>` but no `<author>`. I'm afraid you have to rely on either the XML tagging or the punctuation, and there's no sense in introducing an additional layer of pseudo-markup.

yes, exactly: for DEU <title> element appears to contain the text titles and subtitles for a part of the entries, but quite often not: in these cases just a collection title  https://github.com/distantreading/WG1/issues/40#issuecomment-636718477

**Comment by jberenike** [comment]
> Here's the titles only file for SLV, for comparison (This one has some works with multiple titles)
> [slv-titles.txt](https://github.com/distantreading/WG1/files/4710415/slv-titles.txt)

yes , looks quite different!

**Comment by kmihurko** [comment]
Actually, in these titles the subtitles are already included. I wrote to Tomaž concerning this ... I would prefer to have the difference between the title (Deseti brat) and subtitle (izviren roman) ... because this is not the same as having a genre included in the title (Potresna povest - The tale about the earthquake ...) But this is of course from the perspective of the literary scholar ...

**Comment by RoxanaPatras** [comment]
> On 01/06/2020 09:21, Roxana PATRAS wrote: However, I today it's favors' day: Lou, could you please update the plot for the Rom collection? I think I should add 4 more titles in the annotation table.
> Not sure what you mean. I don't think I should update your annotation table. The mosaic and index.html at https://distantreading.github.io/ELTeC/rom/ were last updated on 30/5 and show a total of 70 titles. If that's wrong, maybe you forgot to check some new titles into the repo?

thanks! I haven't checked that from Saturday :-)).

**Comment by jberenike** [comment]
> I treated "life", "adventures", "roman", "(hi)story", "volume" - embedded in either first part or 2nd part of a title - as genreEntity (see Genette's remarks on the generic context of titles in Seuils) and not as otherEntity.

in my annotation I have adopted this wide notion of genre - including "aufzeichnungen," "geschichte," "bündnergeschichte" etc., but no two or more word units ("sozialer roman"), although this is a bit tricky for languages such as German that like to create ad hoc compounds. So "Sozialroman" would be a genre, but sozialer Roman not. 

However, there is a difference in meaning between ADJ + N ("sozialer roman") and N ("sozialroman"), even if "sozialroman" were a newformation, so I think we're being quite consistent in not splitting up the compounds (in German).

**Comment by RoxanaPatras** [comment]
> > I treated "life", "adventures", "roman", "(hi)story", "volume" - embedded in either first part or 2nd part of a title - as genreEntity (see Genette's remarks on the generic context of titles in Seuils) and not as otherEntity.
> 
> in my annotation I have adopted this wide notion of genre - including "aufzeichnungen," "geschichte," "bündnergeschichte" etc., but no two or more word units ("sozialer roman"), although this is a bit tricky for languages such as German that like to create ad hoc compounds. So "Sozialroman" would be a genre, but sozialer Roman not.
> 
> However, there is a difference in meaning between ADJ + N ("sozialer roman") and N ("sozialroman"), even if "sozialroman" were a newformation, so I think we're being quite consistent in not splitting up the compounds (in German).

You're really lucky in this case because you have only one word for the entire thing! And this is a good argument (for all of us involved in this annotation) to revise the "indicatorEntity" column and include attributes of genre indicators. Thanks!!!.

**Comment by jberenike** [comment]
last note on GenreIndicators: I have treated the default like this.

"Sybille. Eine Selbstbiographie" ('Sybille. A self-biography')

genreIndicatorRole  |  genreIndicatorSyntax
**attribute** | head

Chose attribute b/c the subtitle is an attribution to the main title.

**Comment by CvetanaJK** [comment]
I have gone through the whole Serbian collection (49 works) and corrected the title for ElTEC collcetion to correspond to the first edition (scan if I have it or catalogue data). Subtitles are in the catalogue separated by a semicolon while in the printed edition it was a newline. I put as "contentElementConnection" = interpunc.  Would that be correct?

**Comment by CvetanaJK** [comment]
One more question: I have two works with titles that have three parts. Here is one in English:

Jurmusa and Fatima or Turkish force eats itself : a story about the liberation of six counties 1832-1834
1. part Jurmusa and Fatima
2. Turkish force eats itself
3.   a story about the liberation of six counties 1832-1834

Besides I have two many entities for our columns: two person entities (Jurmusa, Fatima), one palce entity (Turkish), one genreEntity (story), 3 other entities (force, liberation counties), and one date (1832-1834)

**Comment by galleron** [comment]
@CvetanaJK : if the printed edition has a newline, you should have a "no" in the contentElementConnection column. Catalogue builders often normalize such details, they need something to split the information - so they would haven't captured the typographical aspect.
@CvetanaJK I think the two person entities are not a problem. As for the otherEntities (force, liberation, counties), you can add an otherIII column, with the other corelated columns. Finally, please put the dates  at the end of the line (after the "focus"), as a reminder. I guess we do not have many titles with 4 otherEntities, this would be an outlier, probably.
@jberenike : I agree about "A selfbiography" being an attribute and a head - I'd just add that I've marked similar cases as attribute + aposition, when the second part of the title was separated by a coma from the first.
